orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -229,7 +230,8 @@ def _get_session_config(self):
             if self._use_external_gpu_allocator:
                 provider_options = [{"device_id": str(self._device.index),
                                      "gpu_external_alloc": str(self._torch_alloc),
-                                     "gpu_external_free": str(self._torch_free)}, {}]
+                                     "gpu_external_free": str(self._torch_free),
+                                     "gpu_external_empty_cache": str(self._torch_empty_cache)}, {}]
             else:
                 provider_options = [{"device_id": str(self._device.index)}, {}]
```

Comment:
we need to update the default CUDA options for ORTModule... 

pytorch uses cudnn_conv_algo_search=HEURISTIC by default.
cudnn_conv_use_max_workspace should be default true for ORTModule.